Examples/Cxx/Examples.h

```diff
@@ -279,15 +279,15 @@ namespace CIGREMV {
         Int numberPVUnitsPerPlant = numberPVUnits / numberPVPlants;
 
         // PV controller parameters
-        Real scaling_P = 10.0;
-        Real scaling_I = 1000.0;
-
-        Real KpPLL = 0.25/scaling_P;
-        Real KiPLL = 2/scaling_I;
-        Real KpPowerCtrl = 0.001/scaling_P;
-        Real KiPowerCtrl = 0.08/scaling_I;
-        Real KpCurrCtrl = 0.3/scaling_P;
-        Real KiCurrCtrl = 10/scaling_I;
+        Real scalingKp = 1;
```

Comment:
As the change of this parameter for cigre mv is a significant change, mention it in the pull request description